scripts/student_athletes_academic.json

```diff
@@ -0,0 +1,4165 @@
+                    },
+                    {
+                        "name": "fed_rate_female_wh_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "datatab_singleyr_overall_fed_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2011_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2010_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2009_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2008_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2007_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2006_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2005_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2004_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2003_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2002_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2001_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_2000_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_1999_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_1998_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_1997_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_1996_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_1995_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "datatab_singleyr_gender_fed_sb",
+                        "type": "int"
+                    },
+                    {
+                        "name": "fed_rate_male_2011_sb",
+                        "type": "int"
+              
```

Comment:
Add a new line

Reply:
sorry, @henrykironde could you please specify where I should put a new line.
I did the other changes..

Reply:
At  the end of the script

Reply:
check the red icon at line 4165

Reply:
yes, I'll remember that.. thanks.